src/reporters/json.ts

```diff
@@ -154,8 +154,41 @@ function formatTLS(tls: SecurityDetails) {
   };
 }
 
+/**
+ * List of wildcard header keys that should be sanitized/redacted from the request/response headers
+ *
+ * Spec followed by APM agent is used as a reference.
+ * https://github.com/elastic/apm/blob/36a5abd49ff156c80cf0c9e2e1eac919873cb18b/specs/agents/sanitization.md?plain=1#L27
+ */
+const SANITIZE_HEADER_KEYS = [
+  /^password$/i,
```

Comment:
[question]

Is there any reason for not adding all the "default" keys available in the APM document?

<img width="564" alt="default keys" src="https://github.com/elastic/synthetics/assets/15065076/f502b668-15ca-482a-8cfd-38f695e77b58">

Reply:
Mainly because it was an exhaustive set of list and we can introduce these extra headers like credit, card etc incase its required. Also frontend headers differ a lot between the backend transactions which might involve order processing system and other services which makes sense for backend agents.